### DISCUSSION TITLE
Future roadmap and updates

### DISCUSSION THREAD

**Discussion by bhall7** [eroyee/openwebrx_E#8] [General]
Hello,

When Jakob updates the main OpenWebRX repository, how will openwebrx_E get updates? Would any updates need to be manually brought in from Jakob's repo to this one?

I might recommend a "How to Install" explainer or a Wiki entry that tells new users how to get started with openwebrx_E.

Thanks!

**Comment by eroyee** [comment]
If you were using this version of OpenWebRX and you wished to include changes that others had made elsewhere (Michele's work for example) you'd need to merge that into your own code since, at this point, I can't maintain this version much beyond what's there presently.

Just to explain further; for me this always was a limited experiment, not just with the code and feature specific items I had the list for, but also to see if a positive and inclusive/collaborative attitude towards development of an OpenWebRX fork might encourage a similar approach elsewhere. The interest and contribution from several people has been great but unfortunately the work hasn't been taken up in a single unified version. To my mind multiple forks for such a niche 'market' are not the best way of moving forward and can lead to a somewhat fractured outcome at the expense of the community, unless a strong contender with one or more time-rich developer(s) moves to the fore.

Additionally, from a personal time/cost perspective, I had a certain window of opportunity to work on the code and trial the experiment. That window has more or less passed now so, from me, you have what you have. I can't say if I'll do any more in the future or not, but it's pleasing to see a couple of other forks have moved ahead, hopefully they'll keep abreast of things.
